patches/components-content_settings-core-common-content_settings.h.patch

```diff
@@ -1,13 +1,14 @@
 diff --git a/components/content_settings/core/common/content_settings.h b/components/content_settings/core/common/content_settings.h
-index 0565046f47c016b89815d38e54648c2fd42d4e9d..4b699e7d4c169f197a8ca4100b507cb725983d57 100644
+index 0565046f47c016b89815d38e54648c2fd42d4e9d..3928e53c681c0388bc5a30966a06eee51e32fe86 100644
 --- a/components/content_settings/core/common/content_settings.h
 +++ b/components/content_settings/core/common/content_settings.h
-@@ -75,6 +75,8 @@ struct RendererContentSettingRules {
+@@ -75,6 +75,9 @@ struct RendererContentSettingRules {
    ContentSettingsForOneType autoplay_rules;
    ContentSettingsForOneType client_hints_rules;
    ContentSettingsForOneType popup_redirect_rules;
 +  ContentSettingsForOneType fingerprinting_rules;
 +  ContentSettingsForOneType brave_shields_rules;
++  ContentSettingsForOneType speedreader_rules;
```

Comment:
We're avoiding changes on patches when possible, to do that you can do a chromium_src override for `components/content_settings/core/common/content_settings.h` and do a define for the needed changes, then just add in the define constant here.  It'll save for example next time we need to do a patch change.